packages/css/src/components/radio/radio.scss

```
@@ -44,111 +23,181 @@
   gap: var(--ams-radio-gap);
   line-height: var(--ams-radio-line-height);
   outline-offset: var(--ams-radio-outline-offset);
+  position: relative;
+  text-decoration-thickness: var(--ams-radio-text-decoration-thickness);
+  text-underline-offset: var(--ams-radio-text-underline-offset);
+
+  @include text-rendering;
+}
 
-  &:hover {
-    color: var(--ams-radio-hover-color);
-    text-decoration-line: underline;
-    text-decoration-thickness: var(--ams-radio-hover-text-decoration-thickness);
-    text-underline-offset: 0.375rem;
+.ams-radio__icon-container {
+  block-size: var(--ams-radio-icon-container-block-size);
+  display: flex;
+  flex-shrink: 0;
```

Let’s follow this advice from the [specification](https://www.w3.org/TR/css-flexbox-1/#flex-components):

> Authors are encouraged to control flexibility using the `flex` shorthand rather than with its longhand properties directly, as the shorthand correctly resets any unspecified components to accommodate common uses.

```suggestion
  flex: none;
```